Packs/CheckPointHarmonyEndpoint/Integrations/CheckPointHarmonyEndpoint/CheckPointHarmonyEndpoint.py

```diff
@@ -0,0 +1,2703 @@
+
+
+@polling_function(
+    name=demisto.command(),
+    interval=arg_to_number(demisto.args().get("interval", 30)),
+    timeout=arg_to_number(demisto.args().get("timeout", 600)),
+    poll_message="Remediation operation abort request is executing",
+    requires_polling_arg=False,
+)
+def push_operation_abort_command(args: Dict[str, Any], client: Client) -> PollResult:
+    """Aborts the given Remediation Operation.
+       Aborting an operation prevents it from being sent to further Harmony Endpoint Clients.
+       Clients that have already received the operation are not affected.
+
+    Args:
+        client (Client): Harmony API client.
+        args (dict): Command arguments from XSOAR.
+
+    Returns:
+        PollResult: outputs, readable outputs and raw response for XSOAR.
+    """
+    if not args.get("job_id"):
+        remediation_operation_id = args.get("remediation_operation_id", "")
+        SCHEDULED_COMMANDS_MAPPER[demisto.command()].message = (
+            f"Remediation operation {remediation_operation_id} was aborted successfully."
+        )
+
+        response = client.push_operation_abort(remediation_operation_id=remediation_operation_id)
+        args["job_id"] = response.get("jobId")
+
+    return schedule_command(args, client, demisto.command())
+
+
+@polling_function(
+    name=demisto.command(),
+    interval=arg_to_number(demisto.args().get("interval", 30)),
+    timeout=arg_to_number(demisto.args().get("timeout", 600)),
+    poll_message="Anti malware scan request is executing",
+    requires_polling_arg=False,
+)
+def anti_malware_scan_command(args: Dict[str, Any], client: Client) -> PollResult:
+    """Performs an Anti-Malware scan on computers matching the given query.
+
+    Args:
+        client (Client): Harmony API client.
+        args (dict): Command arguments from XSOAR.
+
+    Returns:
+        PollResult: outputs, readable outputs and raw response for XSOAR.
+    """
+
+    if not args.get("job_id"):
+        request_body = build_request_body(args)
+        response = client.anti_malware_scan(request_body)
+        args["job_id"] = response.get("jobId")
+
+    return schedule_command(args, client, demisto.command())
+
+
+@polling_function(
+    name=demisto.command(),
+    interval=arg_to_number(demisto.args().get("interval", 30)),
+    timeout=arg_to_number(demisto.args().get("timeout", 600)),
+    poll_message="Anti malware update request is executing",
+    requires_polling_arg=False,
+)
+def anti_malware_update_command(args: Dict[str, Any], client: Client) -> PollResult:
+    """Updates the Anti-Malware Signature Database on computers matching the given query.
+
+    Args:
+        client (Client): Harmony API client.
+        args (dict): Command arguments from XSOAR.
+
+    Returns:
+        PollResult: outputs, readable outputs and raw response for XSOAR.
+    """
+    if not args.get("job_id"):
+        request_body = build_request_body(args)
+        request_body["operationParameters"] |= {
```

Comment:
You are using `|=`, but what if `operationParameters` is empty?
Make sure to handle this on other places too.

Reply:
it can't be empty cause operationParameters's fields are required or have a default value.